<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrePreferencesForm</class>
 <widget class="QWidget" name="OrePreferencesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>8</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-14</y>
        <width>365</width>
        <height>396</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame1">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout1">
          <property name="margin">
           <number>8</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout1">
            <item>
             <widget class="QLabel" name="encodingLabel">
              <property name="text">
               <string>Audio encoding:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="encodingComboBox"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout2">
          <property name="margin">
           <number>8</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout21">
            <item>
             <widget class="QLabel" name="chooseDirectoryLabel">
              <property name="text">
               <string>Save folder:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="chooseDirectoryButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Choose folder</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="currentDirectoryLabel">
            <property name="text">
             <string>/home/user/MyDocs</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout22">
            <item>
             <widget class="QLabel" name="chooseFileNameFormatLabel">
              <property name="text">
               <string>File name format:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="updateFileNameFormatButton">
              <property name="text">
               <string>Update format string</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout23">
            <item>
             <widget class="QLineEdit" name="fileNameFormatLineEdit"/>
            </item>
            <item>
             <widget class="QComboBox" name="formatSpecifierComboBox">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>16777215</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>%%  The % character</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>%o  Recording number (integer, always increasing)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>%O  Recording number, 6 digits (000001, 000002...)</string>
               </property>
              </item>
<!--
              <item>
               <property name="text">
                <string>%q  Caller's number, if available (441234567890)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>%Q  As %q but &quot;unknown&quot; if number not available</string>
               </property>
              </item>
-->
              <item>
               <property name="text">
                <string>%a  Weekday, abbreviated (Mon, Tue, Wed...)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>%A  Weekday, full (Monday, Tuesday, Wednesday...)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>%d  Day of the month, 2 digits (01-31)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>%e  Day of the month (1-31)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>%m  Month number, 2 digits (01-12)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>%b  Month name, abbreviated (Jan, Feb, Mar...)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>%B  Month name, full (January, February, March...)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>%y  Year, 2 digits (00-99)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>%Y  Year, 4 digits (1900-2100)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>%l  Hour, 12h (1-12)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>%I  Hour, 12h, 2 digits (01-12)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>%k  Hour, 24h (0-23)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>%H  Hour, 24h, 2 digits (00-23)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>%p  AM or FM, uppercase (note reversed case of p and AM/PM)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>%P  am or pm, lovercase (note reversed case of P and am/pm)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>%M  Minute, 2 digits (00-59)</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>%s  Seconds since 00:00:00 1970-01-01 UTC</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>%S  Second, 2 digits (00-59)</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="currentFileNameFormatLabel">
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout3">
          <property name="margin">
           <number>8</number>
          </property>
          <item>
           <widget class="QCheckBox" name="unlimitedCheckBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>No limit to number of recordings</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout3">
            <item>
             <widget class="QLabel" name="maxLimitLabel">
              <property name="text">
               <string>Max number of files</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="fileQuantitySpinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
              <property name="value">
               <number>20</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout4">
          <property name="margin">
           <number>8</number>
          </property>
          <item>
           <widget class="QCheckBox" name="recordOnStartCheckBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Start recording on start-up</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="recordPhoneCheckBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Record phone calls</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>unlimitedCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>fileQuantitySpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>203</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
